packages/components/src/internal/components/entities/actions.ts

```diff
@@ -726,6 +817,44 @@ export function getCrossFolderSelectionResult(
     });
 }
 
+export function getContainersFromSelections(
+    dataRegionSelectionKey: string,
+    dataType: string, // 'samples' | 'exp.data' | 'assay',
+    useSnapshotSelection?: boolean,
+    rowIds?: string[] | number[],
+    picklistName?: string
+): Promise<Record<string, any>[]> {
```

Comment:
are the response objects here `Container[]`? just curious if some other type other than `any` could be used

Reply:
These are really two-field "rows" having a rowId and a container entityId. I can make that more clear with the type.